.github/actions/build-docker/action.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Build Quickwit docker image'
+description: 'Build & push docker image'
+inputs:
+  version:
+    description: 'Version'
+    required: false
```

Comment:
Not sure why version is not required

Reply:
For production build we don't specify `version` in order to have an image `quickwit/quickwit` as opposed to `quickwit/quickwit-nightly`